sky/tools/sky_utils.py

```diff
@@ -84,7 +84,7 @@ def _contains_duplicates(strings):
 
 def _is_macho_binary(filename):
   """Returns True if the specified path is file and a Mach-O binary."""
-  if not os.path.isfile(filename):
+  if os.path.islink(filename) or not os.path.isfile(filename):
```

Comment:
Note: just like `-f` checks in shell script, `os.path.isfile` will return `True` when passed the path to a symlink that resolves to a file.